Major overhaul of contribution issue templates to reflect new processes & governance (#2)
TheJuanAndOnly99/community: .github/ISSUE_TEMPLATE/Special-Interest-Group-Contribution.md

- [ ] I confirm I am contributing on behalf of a FINOS member (`add FINOS member organization name`) or I will seek a maintainer from a FINOS member during socialization 

## Understading the contribution Process
Completing an onboarding of a special interest group into FINOS comprises following 5 main steps described in details in our [Community Repository](https://community.finos.org/docs/governance/Software-Projects/contribution#contribution-of-an-existing-code-base-into-finos-as-a-new-project). 

 - [ ] I confirm I have reviewed the contribution process.

## Understanding FINOS

As a SIG onboarding into FINOS, you should familiarize yourself and your contributor team with the following FINOS information: [FINOS overview](https://www.finos.org/hubfs/An%20Introduction%20to%20FINOS.pdf), [FINOS SIG Governance](/docs/governance/#special-interest-groups)), [FINOS Maintainers cheatsheet](https://community.finos.org/docs/finos-maintainers-cheatsheet/).

- [ ] I confirm I am familiar with the FINOS governance and maintainers (Chair) responsibilities

---
# SIG information

## Describing The Contribution

> This section contains information to be provided by the contributor to give it the best chances to be approve stage. A SIG is a great place to have conversation around a certain area of interest, generally in the "problem space" to then proceed to create or contribute a software or standard project to provide a solition. Please review a list of [existing SIGs](https://community.finos.org/docs/governance/special-interest-groups/) before proposing one as you might find your area of interest already covered. 

Reviewer: ```suggestion
This section contains information to be provided by the contributor to give it the best chances to be approved. A SIG is a great place to have conversation around a certain area of interest, generally in the "problem space" to then proceed to create or contribute a software or standard project to provide a solition. Please review a list of [existing SIGs](https://community.finos.org/docs/governance/special-interest-groups/) before proposing one as you might find your area of interest already covered. 
```

|Name                        |Affiliation              |Work Email Address             |Github / GitLab username              |
|----------------------------|-------------------------|-------------------------------|--------------------------------------|
|Contributor1 Example        |Example LTD              |con1@example.com               |@con1xampleabc                        |
|Contributor2 Example        |Example LTD              |con2@example.com               |@con2examplexyz                       |

### Target Participants
*Describe the participant profile (background, position, etc) and if applicable organizational profile (financial institutions, tech vendors, consulting firms, etc) you would like to get contributions from*

## Preferred SIG infrastructure
If the SIG is approved, FINOS will provision the infrastructure necessary for [collaboration](https://community.finos.org/docs/collaboration-infrastructure). Below a list of questions which will help us accelerate the onboarding process and tailor it to the maintainers needs:

> - Which hosting platform does the project prefer? *[Github](https://github.com/finos/)|[GitLab](https://gitlab.com/finos)* 

Reviewer: Same comment as above applies see https://github.com/TheJuanAndOnly99/community/pull/2/files#r2098409330

> - Please check below which asynchronous collaboration channels will the project like to use?

Reviewer: ```suggestion
- Please check below which asynchronous collaboration channels the project would like to use:
```

  - [ ] GitHub Issues
  - [ ] GitHub Public Discussions

>   - [ ] GitHub Team Discussions _(consisting of the above described contributors)_

Reviewer: Same as above. GitHub Teams Discussions has been deprecated.

    - Public (Mandatory) _(consisting of the above described contributors)_
    - [ ] Private (Optional)

>   - [ ] Mailing-list (lists.finos.org)

Reviewer: Same comment as for the software project contribution.

- [ ] FINOS Leadership confirmed timing and procedure of Governing Board vote with contributors

### Governing Board Approval findings 
- [ ] FINOS Governing Board reviewed the contribution and the contribution was: `accepted|rejected`

> FINOS team to enter high level findings summary here supporting the decision

## 3. Preparing For Onboarding
Before the FINOS team can onboard your project, there are a few housekeeping items that need to be taken care of from an IP and infastructure standpoint. These must be completed by the contributor, with help if required from the FINOS Infrastructure team, so please familiarize yourself with the [requirements](/docs/governance/software-projects/contribution/#step-4-ip-review--transfer-of-contributions).

### Steps
- [ ] Usable name and logo identified (either existing or new)
  - [ ] FINOS team confirmed project brand and logo are usable in the public domain. This might require completion of [project contribution agreement](https://community.finos.org/governance-docs/The.Linux.Foundation.--.Form.of.Trademark.Assignment.20221202.pdf) to properly assign trademarks.
  - [ ] Failed the above, new project name and logo is identified

> - [ ] FINOS team enabled documentation DCO (developer certificate of origin) to enforce CC-by-40 license, [default for SIGs](/docs/governance#special-interest-groups)

Reviewer: DCO cannot be enabled until after we control the repo. As such this should happen after the code transfer.

   - [ ] FINOS agrees with Chairs on category and sub-category (for [FINOS Landscape](https://landscape.finos.org/) placement
   groups)

## 4.  Transfer and onboarding in FINOS
This is performed by FINOS Infra once the three previous stages are complete, with support from the contributor and the FINOS Infra team.

## Code transfer preparation
- [ ] Maintainer has made [finos-admin](http://github.com/finos-admin) Admin of the GitHub repository to transfer

## Existing assets transfer Meeting
- [ ] Transfer all assets as GitHub repositories under github.com/finos
- [ ] Rename main branch to `main` (instead of `master`)
- [ ] Check that CVE (and preferably static code analysis, if applicable) scanning is in place
- [ ] Enable [branch protection](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches) (`Require a pull request before merging`)
- [ ] Enable automated dependency update, using [Renovate](https://renovatebot.com/)

> - [ ] (best effort or to be completed right after onboarding) Update release coordinates and code namespace to include `finos`

Reviewer: ```suggestion

```

SIGs should not be doing releases

- [ ] Communication channels are linked front and center in the project README.md

> - [ ] Communication channels are linked front and center in the project README.md

Reviewer: ```suggestion

```
Delete duplicate line
